element_interface/utils.py

```diff
@@ -146,25 +147,24 @@ def ingest_csv_to_table(
             )
 
 
-def recursive_search(key: str, dictionary: dict) -> any:
-    """Return value for key in a nested dictionary
-
-    Search through a nested dictionary for a key and returns its value.  If there are
-    more than one key with the same name at different depths, the algorithm returns the
-    value of the least nested key.
-
-    Args:
-        key (str): Key used to search through a nested dictionary
-        dictionary (dict): Nested dictionary
+def insert1_skip_full_duplicates(table, entry):
```

Comment:
Awesome. This will be integrated into DataJoint shortly. https://github.com/datajoint/datajoint-python/issues/1049